### DISCUSSION TITLE
because this doesn't work

### DISCUSSION THREAD

**Discussion by clopezpro** [nitrojs/nitro#2715] [Q&A]
![image](https://github.com/user-attachments/assets/4b42d77c-4faa-4f1f-9f15-d7de4436bb9f)

Generate erro ```~/server/controller/migration/1.1.0
Cannot find package '~' imported from G:\VUE\facturador\.nuxt\dev\index.mjs```

but if it goes that way it works
![image](https://github.com/user-attachments/assets/62deb512-f67b-420a-a18c-845719cbd0de)

**Comment by stafyniaksacha** [comment]
you may have to use `import('..../${version}')` without using a variable (rute) as shown here: https://vitejs.dev/guide/features.html#dynamic-import
